backend/src/parser/TheParser.ts

```diff
@@ -54,14 +57,25 @@ export default class TheParser {
     }
 
     private parseChildNodes(doc: ChildNode[]): ParseResult {
-        return doc.reduce((p, c) => {
-            const res = this.parseNode(c);
+        const p = {text: '', mentions: [], urls: [], images: []};
+        let prevBQ = false; // if previous node was blockquote
+        for (let node of doc) {
+            if (prevBQ && node.type === 'text') {
+                // remove a single newline after blockquote, allow only a single one if multiple were present
+                node = {
+                    ...node,
+                    data: node.data.replace(/^(\r?\n|[\r\n])(\r?\n|[\r\n])?(\r?\n|[\r\n])*/, '$2')
```

Comment:
I think this *or* condition is redundant here: `|[\r\n]`, and if I change pattern to 
```js
data: node.data.replace(/^(\r?\n)(\r?\n)?(\r?\n)*/, '$2')
```
tests are passing too

Reply:
Sure, but it doesn't account for `\r` line break type (not sure if we need to support it, but we do in other places).
https://en.wikipedia.org/wiki/Newline

Reply:
And now that I looked into it, my regex doesn't account for `\n\r` 🤷‍♂️

Reply:
> Sure, but it doesn't account for \r line break type

maybe add tests with `\r` then

Reply:
> Sure, but it doesn't account for \r line break type 

maybe add tests with `\r` then